app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -193,6 +307,13 @@ fun SettingsScreen(
         SettingHydrationNotificationToggle(model)
 
         SettingCupVolumeMl(model)
+
+        CategoryRow(
+            icon = Icons.Outlined.Snowboarding,
+            title = "Theme"
```

Comment:
Maybe `Appearance` instead of `Theme`.
And [Style](https://mui.com/material-ui/material-icons/?theme=Outlined&query=style&selected=StyleOutlined) icon, if it exists in Android.